injected/src/content-scope-features.js

```diff
@@ -41,13 +41,22 @@ export function load(args) {
         injectName: import.meta.injectName,
     };
 
-    const featureNames = typeof importConfig.injectName === 'string' ? platformSupport[importConfig.injectName] : [];
+    const bundledFeatureNames = typeof importConfig.injectName === 'string' ? platformSupport[importConfig.injectName] : [];
 
-    for (const featureName of featureNames) {
-        const ContentFeature = platformFeatures['ddg_feature_' + featureName];
-        const featureInstance = new ContentFeature(featureName, importConfig, args);
-        featureInstance.callLoad();
-        features.push({ featureName, featureInstance });
+    // prettier-ignore
+    const featuresToLoad = isGloballyDisabled(args)
+        // if we're globally disabled, only allow `platformSpecificFeatures`
+        ? platformSpecificFeatures
+        // otherwise, use `enabledFeatures` if available, falling back to every feature in the bundle
+        : args.site.enabledFeatures || bundledFeatureNames;
```

Comment:
When don't we have `enabledFeatures`?

Reply:
when the extension calls 'load'

Reply:
I wonder if we should make it more explicitly commented?